### PR TITLE
Add documentation for ivy

### PR DESCRIPTION
**Pull request by Somelauw**
It didn't have any documentation yet

### CODE REVIEW

File: layers/+completion/ivy/README.org [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+* Install
+To use this configuration layer, add it to your =~/.spacemacs=. You will need to
+add =ivy= to the existing =dotspacemacs-configuration-layers= list in this
+file.
+
+* Configuration
+You can customize ivy with the following variables:
+
+1. ivy-wrap Whether C-n and C-p should wrap-around when at the the first or last candidate. The default value is nil.
```
Comment (sdwolf) [inline layers/+completion/ivy/README.org]: This list looks a little weird, maybe wrap the variable names inside equal signs: `=ivy-wrap=` to format them or matbe turn this list into a table, whatever you feel looks best.

File: layers/+completion/ivy/README.org [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+bindings]]. Some useful key bindings are presented in following table.
+
+| Key Binding | Description                                                                                          |
+|-------------+------------------------------------------------------------------------------------------------------|
+| ~RET~       | call default action on current candidate                                                             |
+| ~M-RET~     | the same as ~RET~ but doesn't close completion minibuffer                                            |
+| ~tab~       | complete partially                                                                                   |
+| ~M-o~       | show the list of valid actions on current candidate (then press any of described keys to execute it) |
+| ~C-M-o~     | the same as ~M-o~ but doesn't close completion minibuffer                                            |
+| ~C-'~       | use avy to quickly select completion on current page (sometimes faster than using arrows)            |
```
Comment (sdwolf) [inline layers/+completion/ivy/README.org]: Can you please add the "C-M-j" keybinding documentation here as well, I remember an issue from a few weeks ago regarding it's functionality and a discussion to add it to the documentation.

**Comment by Somelauw** [comment]
@sdwolf thanks, I have added both suggestions

**Comment by syl20bnr** [comment]
Thank you ! 💜 
Cherry-picked into develop branch, you can safely delete your branch.

**Pull request closed without merging**